docker/diffkemp-devel/run-container.py

```diff
@@ -0,0 +1,39 @@
+import subprocess
```

Comment:
```suggestion
#!/bin/env python3
import subprocess
```
+ make this file executable.